debezium-server-name-mapper/README.md

```diff
@@ -0,0 +1,77 @@
+# Custom Topic Name Mapper
+
+This example demonstrates how to develop and deploy a bean that would allow the developer to implement custom topic naming policy.
```

Comment:
```suggestion
This example demonstrates how to develop and deploy a bean that would allow the developer to implement a custom topic naming policy.
```

Reply:
Should also mention that this is specific to Debezium Server, including a link to its docs.